package.json

```diff
@@ -10,7 +10,9 @@
     "url": "https://github.com/ministryofjustice/prepare-a-case.git"
   },
   "scripts": {
-    "start": "IP=$(ipconfig getifaddr en0) && /usr/X11/bin/xhost + $IP && DISPLAY=$IP:0 docker compose up",
+    "start": "run-script-os",
```

Comment:
I like it but run-script-os is going to run outside of docker, which defeats the point of the docker work to remove node being a stack requirement.

However given they will have some npm version to run these shortcuts, I would say use npx run-script-os and remove the entry from package.json. Defo don't want it in package.json

PS) unsure if start:detached will still work.

PPS) Update the readme with the mac xquartz thingy.